product_docs/docs/postgres_distributed_for_kubernetes/1/known_issues.mdx

```diff
@@ -0,0 +1,63 @@
+---
+title: 'Known issues and limitations'
+---
+
+These known issues and limitations are in the current release of EDB Postgres Distributed for Kubernetes. 
+
+## Upgrades 
+
+This version of EDB Postgres Distributed for Kubernetes does **not support** migrating from existing Postgres databases, nor upgrading 
+any of its components to more recent image versions.
+
+## Data encryption 
+
+EDB Postgres Distributed for Kubernetes does not support enabling [Transparent Data Encryption](/tde/latest/) (TDE) to encrypt user data of EDB Postgres Advanced 
+and EDB Postgres Extended databases. 
```

Comment:
This is going to be added very soon, pre-GA. I'd take it out at the moment, and we can check later in the week but I think we won't need it at all.